src/cascadia/TerminalApp/App.cpp

```diff
@@ -231,6 +233,18 @@ namespace winrt::TerminalApp::implementation
                 profileMenuItem.Icon(_GetIconFromProfile(profile));
             }
 
+            if (profile.GetGuid() == defaultProfileGuid)
+            {
+                // Contrast the default profile with others in font weight.
+
+                // The "bold" and "normal" font weights are 700 and 400 respectively,
+                // so we add 300 to get bold from normal.
+                // If the default font weight is higher, so is the accented one (but no higher than 999).
+                const uint16_t defaultFontWeight = profileMenuItem.FontWeight().Weight;
+                const uint16_t accentedFontWeight = gsl::narrow<uint16_t>(defaultFontWeight + 300u);
+                profileMenuItem.FontWeight({ std::min(gsl::narrow<uint16_t>(999u), accentedFontWeight) });

```

Comment:
we shouldn't need the narrow on 999u; it is formally proven that it's within uint16_t :)

Reply:
I have to convert it statically to uint16_t in order to get to the uint16_t version of std::min.  Of course I could be using static_cast here, but gsl::narrow does not harm and is constexpr.